pkg/humio/client_mock.go

```diff
@@ -215,6 +216,34 @@ func (h *MockClientConfig) DeleteRepository(config *humioapi.Config, req reconci
 	return nil
 }
 
+func (h *MockClientConfig) AddUser(config *humioapi.Config, req reconcile.Request, hu *humiov1alpha1.HumioUser) (*humioapi.User, error) {
+	h.apiClient.User = humioapi.User{
+		Username:    hu.Spec.Username,
+		ID:          kubernetes.RandomString(),
+		FullName:    hu.Spec.FullName,
+		Email:       hu.Spec.Email,
+		Company:     hu.Spec.Company,
+		CountryCode: hu.Spec.CountryCode,
+		Picture:     hu.Spec.Picture,
+		IsRoot:      hu.Spec.IsRoot,
+		CreatedAt:   hu.Spec.CreatedAt,
```

Comment:
```suggestion
```

Reply:
This is not used by the AddUserV2 mutation that is called by this, so this doesn't do anything, so no need to have CreatedAt included here.